backend/src/main/java/com/bakdata/conquery/models/messages/network/specific/ForwardToWorker.java

```diff
@@ -12,40 +14,66 @@
 import com.bakdata.conquery.models.worker.Worker;
 import com.bakdata.conquery.util.io.ConqueryMDC;
 import com.bakdata.conquery.util.progressreporter.ProgressReporter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.SneakyThrows;
 
 @CPSType(id="FORWARD_TO_WORKER", base=NetworkMessage.class)
-@RequiredArgsConstructor @Getter
+@RequiredArgsConstructor(access = AccessLevel.PROTECTED) @Getter
 public class ForwardToWorker extends MessageToShardNode implements SlowMessage {
 
+	private static final ObjectWriter WRITER = Jackson.BINARY_MAPPER.copy().writerFor(WorkerMessage.class).withView(InternalOnly.class);
+
+	@SneakyThrows // TODO check if this can kill whole threads.
```

Comment:
Ich glaube nicht.

Reply:
mir gehts mehr um die threads in denen es fliegen könnte, aber ich habe noch keine JsonProcessingException beim schreiben erlebt ehrlich gesagt